lib/WebServer/LapTriggerWebServer.cpp

```diff
@@ -150,13 +150,35 @@ void LapTriggerWebServer::webSocketEvent(uint8_t clientId, WStype_t type, uint8_
 
         for (index = 0; index < length; ++index)
         {
-            cmd += reinterpret_cast<char *>(payload)[index];
+            char temp = reinterpret_cast<char *>(payload)[index];
```

Comment:
Move now the whole parsing part to a separate function. This is too much now for just one.